pkg/landscaper/controllers/deployitem/controller.go

```diff
@@ -209,8 +210,10 @@ func (con *controller) detectPickupTimeouts(log logging.Logger, di *lsv1alpha1.D
 	return &requeue, nil
 }
 
-func (con *controller) detectAbortingTimeouts(log logging.Logger, di *lsv1alpha1.DeployItem) (*time.Duration, error) {
-	logger := log.WithValues("operation", "DetectAbortingTimeouts")
+func (con *controller) detectAbortingTimeouts(ctx context.Context, di *lsv1alpha1.DeployItem) (*time.Duration, error) {
+	logger, _ := logging.FromContextOrNew(ctx, nil)
+	logger = logger.WithValues(lc.KeyMethod, "detectAbortingTimeouts")
```

Comment:
```suggestion
	logger, ctx := logging.FromContextOrNew(ctx, nil, lc.KeyMethod, "detectAbortingTimeouts")
```

Reply:
Again the `KeyMethod` logging key, which is only for the current method, but should not be passed in a context to sub methods.